contrib/gorilla/mux/option.go

```diff
@@ -103,3 +109,36 @@ func WithResourceNamer(namer func(router *Router, req *http.Request) string) Rou
 		cfg.resourceNamer = namer
 	}
 }
+
+// WithHeaderTags enables setting of request header as tags on the span
```

Comment:
It does not enable setting. It sets them. We must also warn users here.
```suggestion
// WithHeaderTags specifies that the integration should attach HTTP request headers as
// tags to spans.
// Warning: using this feature can risk exposing sensitive data such as authorisation tokens
// to Datadog.
```

Reply:
Same?